src/scaict_uwu/libs/services/service_container.py

```
@@ -0,0 +1,87 @@
+        self._service_instantiators[name] = instantiator
+
+    # Service functions
+
+    def create_service(self, /, name: str):
+        if not self.has_instantiator(name=name):
+            sys.exit("NoSuchServiceException $name")
+
+        if name in self._services_being_created:
+            sys.exit(
+                "RecursiveServiceDependencyException "
+                + "Circular dependency when creating service!"
```

Ruff (0.11.9)

Explicitly concatenated string should be implicitly concatenated

(ISC003)